warehouse/models/mart/gtfs/fct_monthly_route_service_by_timeofday.sql

```diff
@@ -4,17 +4,30 @@ WITH dim_gtfs_datasets AS (
     SELECT * FROM {{ ref('dim_gtfs_datasets') }}
 ),
 
+-- keep the most recent of month_last_day
+feeds AS (
+    SELECT
+        gtfs_dataset_key as gtfs_key,
+        MAX(LAST_DAY(date, MONTH)) as max_date
+    FROM {{ ref('fct_daily_schedule_feeds') }}
+    WHERE date = LAST_DAY(date, MONTH)
```

Comment:
This appears to filtering results to the last full month (February 29th in this case, currently excluding March), is that the goal here?

Reply:
Yes! I want to filter out March until the whole month of March is ready to be aggregated (rather than aggregate 2 weeks of March and having a row for it)